Bottom: 1432103a64b1b532e30b749c3b166b71ee3d091c
Top:    94f5561cb7e993159d0f011ab046ad27e68ee1a3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-16 00:57:46 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
